app/views/kalibro_ranges/_form.html.erb

```diff
@@ -8,12 +8,16 @@
         <%= render partial: 'no_readings' %>
         <% else %>
         <div class="field-container">
-          <%= f.label :reading_id, 'Reading', class: 'control-label' %>
+          <%= f.label :reading_id, t('reading'), class: 'control-label' %>
           <%= f.select( :reading_id, readings_options(@readings), {class: 'form-control'} ) %>
         </div>
         <div class="help-container">
           <p>
-            The <%= link_to 'Reading', tutorials_path('keywords', anchor: 'reading')%> associated with this <%= link_to 'Range', tutorials_path('keywords', anchor: 'range')%>.
+            <%= t('activemodel.hints.kalibro_range.reading',
```

Comment:
Now we have the ```t_hint``` helper. In the future using it on all the forms will be a feature.